### DISCUSSION TITLE
[functional-parameters] ignorePattern isn't matching against reduce function callbacks

### DISCUSSION THREAD

**Discussion by garethallenstringer** [jonaskello/eslint-plugin-functional#313] [Q&A]
Having difficulty matching `ignorePattern` to a `.reduce()` function for the case of:

`"functional/functional-parameters": ["warn", { "enforceParameterCount": "exactlyOne", "ignorePattern": ["(?<=\\.reduce\\()(.*)(?=\\))"] }]`

in my eslintrc.json rules. I have tried a multitude of other regular expressions but haven't had any joy.

**Comment by RebeccaStevens** [comment]
Unfortunately, regex patterns have there limit.

The proposal in #244 should allow for a nicer way to solve this issue.
